### ISSUE TITLE
RekognitionFacesPolicy seems to take unnecessary parameter

### ISSUE THREAD

**Issue by 53ningen** [awslabs/serverless-application-model#974]
**Description:**

Despite Both `rekognition:CompareFaces` and `rekognition:DetectFaces` actions are not related with Rekognition face collection, [policy template `RekognitionFacesPolicy`](https://github.com/awslabs/serverless-application-model/blob/develop/samtranslator/policy_templates_data/policy_templates.json#L1051-L1079) takes `CollectionId` parameter.

I would like to use policy template `RekognitionFacesPolicy`, but it does not work well because it makes unnecessary constraints for resources.

I suspect this is a bug.

**Questions:**

Why policy template `RekognitionFacesPolicy` takes `CollectionId` parameter?

**Related Document:**

[1] Amazon Rekognition API Permissions: Actions, Permissions, and Resources Reference - Amazon Rekognition
https://docs.aws.amazon.com/rekognition/latest/dg/api-permissions-reference.html
```
API Operations |	Required Permissions   	| Resources
--------------------------------------------------------
CompareFaces   | rekognition:CompareFaces |	None used.
DetectFaces    | rekognition:DetectFaces	| None used.
```

**API References:**

[2] CompareFaces - Amazon Rekognition
https://docs.aws.amazon.com/rekognition/latest/dg/API_CompareFaces.html#API_CompareFaces_RequestSyntax

[3] DetectFaces - Amazon Rekognition
https://docs.aws.amazon.com/rekognition/latest/dg/API_DetectFaces.html#API_DetectFaces_RequestSyntax

**Comment by keetonian** [comment]
You are correct. `CompareFaces` and `DetectFaces` have no required resources in https://docs.aws.amazon.com/rekognition/latest/dg/api-permissions-reference.html. 

That parameter to that policy template can be safely removed.

**Comment by 53ningen** [comment]
Hi, keetonian 
Thank you for your response.

I will try to make pull request to resolve this issue

**Comment by 53ningen** [comment]
Hi, @keetonian 
I created a pull request to resolve this issue.

Could you please review it when you have time?
Thanks.

**Comment by jlhood** [comment]
@53ningen Appreciate the fix! We'll make sure the docs get updated once this change gets released.

**Comment by jlhood** [comment]
Pending release in v1.14.0

**Comment by ShreyaGangishetty** [comment]
Closing this issue as this is released in v1.14.0
